public/keymaps/d/durgod_k320_default.json

```
@@ -0,0 +1,24 @@
+{
+  "keyboard": "Durgod K320",
+  "keymap": "default_537b761",
```

`keyboard` should be the QMK path name for the keyboard, rather than the "marketing" name.
`keymap` can be `default`.

```suggestion
  "keyboard": "durgod/k320",
  "keymap": "default",
```